crates/forge_analyzer/src/definitions.rs

```diff
@@ -1006,15 +1038,34 @@ impl FunctionAnalyzer<'_> {
                     && Some(&ImportKind::Default)
                         == self.res.is_imported_from(def, "@forge/api") =>
             {
+                let first_arg = first_arg?;
+                let is_as_app = authn.first() == Some(&PropPath::MemberCall("asApp".into()));
+
                 let function_name = if *last == "requestJira" {
-                    IntrinsicName::RequestJira
+                    // Resolve Jira API requests to either JSM/JS/Jira as all are bundled within requestJira()
+                    match first_arg {
+                        Expr::TaggedTpl(TaggedTpl { tpl, .. }) => {
+                            if let Some(TplElement { raw, .. }) = tpl.quasis.first() {
+                                resolve_jira_api_type(raw).unwrap_or_else(|| {
+                                    // Conservatively assume any of Jira APIs may be used if we can't statically determine which one
+                                    warn!("Falling back to any Jira request");
+                                    IntrinsicName::RequestJiraAny
+                                })
+                            } else {
+                                panic!("No url identifiable to classify requestJira() type");
+                            }
+                        }
+                        _ => {
```

Comment:
You should also handle the case for when `first_arg` is a string literal: https://rustdoc.swc.rs/swc_ecma_ast/enum.Expr.html#variant.Lit.